Lib/fontTools/feaLib/builder.py

```diff
@@ -1106,7 +1106,13 @@ def set_language(self, location, language, include_default, required):
         if (language == "dflt" or include_default) and lookups:
             self.features_[key] = lookups[:]
         else:
-            self.features_[key] = []
+            # if we aren't including default we need to manually remove the
+            # default lookups, which were added to all declared langsystems
+            # as they were encountered (we don't remove all lookups because
+            # we want to allow duplicate script/lang statements;
+            # see https://github.com/fonttools/fonttools/issues/3748
+            cur_lookups = self.features_.get(key) or []
```

Comment:
did you mean

```suggestion
            cur_lookups = self.features_.get(key, [])
```

Reply:
I didn't mean this, exactly, but happy to make the change if it's more idiomatic :)

Reply:
Well yes it is be more idiomatic, the second parameter to dict.get is the default value when the key is missing, if you don't provide a default to get() you get a None